src/i18n/locales/en/translation.json

```diff
@@ -289,44 +289,45 @@
       },
       "freeze_utxos": {
         "text_primary_button": "Freeze UTXOs",
-        "text_primary_button_all_frozen": "Next",
+        "text_primary_button_all_frozen": "Review bond configuration",
         "text_primary_button_error": "Try again",
         "text_secondary_button": "Cancel",
-        "description_selected_utxos": "Selected UTXOs",
-        "description_unselected_utxos": "The following UTXOs will not be used for the fidelity bond.",
-        "description_selected_utxos_to_freeze": "They will be frozen to remain in jar #{{ jar }}. You can unfreeze them anytime after creating the fidelity bond."
+        "description_selected_utxos": "Selected UTXOs:",
+        "description_unselected_utxos": "The following UTXOs will not be used for the fidelity bond:",
+        "description_selected_utxos_to_freeze": "They will be frozen to remain in Jar #{{ jar }}. You can unfreeze them anytime after creating the bond."
       },
       "review_inputs": {
         "text_primary_button": "Create fidelity bond",
         "text_primary_button_unsafe": "Create fidelity bond with potentially less private UTXOs",
```

Comment:
I'd love to come up with something clever that replaces "potentially less private" without saying "non-cjout UTXOs" 🤣 

---

Just spitballing here, but we could rename the `cjout` label to `jam` or `jammed` (hmm... "jammed" has a negative connotation, so probably don't use that) so we could call "bad" UTXOs "non-jam" or something like that.

Sticking with the jam (and making jam) metaphor, we could call "bad" UTXOs "lumpy" or "clumpy", and "good" ones (mixed, `cj-out`) "smooth" or "silky" or "kosher" or something. "Smooth" is probably best, since it is used in cooking, as in "blend until smooth."

Reply:
Again: this is just a thinking-out-loud kind of thing, not saying we should absolutely do this. I'd also be happy with less playful language, i.e. revolving around privacy, or even the "default" lingo of mixed and unmixed (which works well with the jam metaphor too).

Reply:
I like the idea of inverting it. Highlight the non-jammed UTXOs as bad instead of the other way around. Naming is tricky. _Jammed_ would be the ideal word but you're right -- it's too negative. 🤔 I wouldn't go too far with the lingo. With something like "smooth", "silky" I feel like it might be unclear what we mean unless we also use it in lots of other places in the app...

Reply:
We could also use the "pre-mix" and "post-mix" differentiation, and thus we could say something like:

> ⚠️  Create fidelity bond with pre-mix UTXOs

Reply:
I'm a bit hesitant with using "mixing" as well. IIRC we once said to avoid it since it might be confused with the old school coin mixers from back in the days. I don't have a better suggestion though so until we come up with something better it's definitely better than "potentially less private" imo.